torchci/pages/metrics.tsx

```diff
@@ -300,6 +300,125 @@ export function TimeRangePicker({
   );
 }
 
+/**
+ * Allows the user to pick the TTS metrics.
+ */
+export function TtsPercentilePicker({
+  ttsPercentile,
+  setTtsPercentile,
+}: {
+  ttsPercentile: number;
+  setTtsPercentile: any;
```

Comment:
one easy way to see what the type is just to hover over this where you initialize it and copy it over here.